test.config.js

```diff
@@ -43,3 +45,7 @@ try {
 require("raf/polyfill")
 require("should")
 require("./src/lib/jade_hook")
+
+// Used by relay
+sd.METAPHYSICS_ENDPOINT = "http://localhost:5003"
```

Comment:
Interesting that we haven't tested a relay-based component yet in force

Reply:
Ohhh, that's interesting. I was wondering why this was needed, but I hadn't thought of it from that angle.

Reply:
We actually have been, but the new hooks setup in `SystemContextProvider` seemed to cause an issue, we have tests that were x'd 3 months back to handle the same problem.